app/views/shared/_header.html.erb

```diff
@@ -7,7 +7,7 @@
       <% end %>
       <nav id="menu" class="col-sm" role="menu">
         <%= link_to "About","#{root_url}about", :class => "btn btn-primary about", :role => "menuitem" %>
-        <%= link_to "How to Submit", "#{root_url}how-to-submit", :class => "btn btn-primary how-to-submit", :role => "menuitem" %>
```

Comment:
As was identified by @hectorcorrea, this must remain `#{root_url}how-to-submit` for pages other than the home dashboard (please see https://pdc-describe-staging.princeton.edu/describe/)